src/hooks/useOnboardingCourse.tsx

```diff
@@ -36,17 +36,18 @@ export const OnboardingCourseContextProvider = ({
 
   const [storedDidLaunchResult, storeDidLaunch] = useAsyncStorage(
     `${activeProject?.id}-didLaunchOnboardingCourse`,
+    !!activeProject?.id,
   );
   const [didLaunchCourse, setDidLaunchCourse] = useState<boolean | undefined>(
     undefined,
   );
 
-  const isLoading = isAppConfigLoading || storedDidLaunchResult.isLoading;
-  const isFetched = isAppConfigFetched && storedDidLaunchResult.isFetched;
+  const isLoading = isAppConfigLoading;
+  const isFetched = isAppConfigFetched;
```

Comment:
Should we check to make sure `didLaunchCourse` should be populated here? In theory it seems like the async storage being async could make this hook return `isLoading: false, isFetched: true` before the value is actually loaded.

Reply:
Coming up in the next PR.